ci/test_plugins.rb

```diff
@@ -155,7 +165,9 @@ def install_gem(gem_file)
                 "logstash-filter-memcached", "logstash-filter-mutate", "logstash-filter-prune", "logstash-filter-ruby",
                 "logstash-filter-sleep", "logstash-filter-split", "logstash-filter-syslog_pri", "logstash-filter-translate",
                 "logstash-filter-truncate", "logstash-filter-urldecode", "logstash-filter-useragent", "logstash-filter-uuid",
-                "logstash-filter-xml"],
+                "logstash-filter-xml",
+                ["logstash-filter-elastic_integration", "git@github.com:elastic/logstash-filter-elastic_integration.git"]
```

Comment:
At this point we're having too many aspects or dimensions associated to each plugin, and having a data structure that is organized by dimension is becoming harder to manage, as seen by this new dimension (plugin github org), where we now have to parse if it's a single string or an array with more information in it.

My suggestion would be to change this to a simple list:

```ruby
[
  {
     name: "logstash-codec-avro",
     support: "tier1",
     type: "codec"
  },
  {
     name: "logstash-input-rss",
     support: "unsupported",
     type: "input",
     org: "logstash-plugins"
  },
  {
     name: "logstash-integration-elastic_enterprise_search",
     support: "tier1",
     type: "integration",
     org: "elastic"
  }
]
```

This list can either use simple hashes or Structs. 
This way you can filter for the property that you need, and the org is treated like any other property without requiring special logic.

Reply:
Agree, it would simplify a lot also the navigation. I'll rework it.